src/core/radio/trel_interface.cpp

```diff
@@ -363,7 +363,7 @@ Error Interface::Send(const Packet &aPacket, bool aIsDiscovery)
     Error error = kErrorNone;
     Peer *peerEntry;
 
-    VerifyOrExit(mInitialized && mEnabled, error = kErrorAbort);
+    VerifyOrExit(mInitialized && mEnabled, error = kErrorInvalidState);
```

Comment:
I don't think the change of `error` here will do what we intend (won't change the log).

The `Interface::Send()`  is called from `Link::BeginTransmit()` which checks the error and then calls  `InvokeSendDone(kErrorAbort)` with "Abort" error:

https://github.com/openthread/openthread/blob/fc307401a6aee332b36a94816033c8884ed283c5/src/core/radio/trel_link.cpp#L214-L216

Reply:
oh, I've missed that.